python/tchannel/tornado/message_factory.py

```diff
@@ -45,6 +53,9 @@ def __init__(self, remote_host=None, remote_host_port=None):
         self.remote_host = remote_host
         self.remote_host_port = remote_host_port
 
+        self.in_checksum = {}
+        self.out_checksum = {}
```

Comment:
Consider adding a comment (not a docstring) about what these dictonaries will hold.